tests/selftest_cli.js

```diff
@@ -0,0 +1,28 @@
+const assert = require('assert').strict;
+const path = require('path');
+const child_process = require('child_process');
+
+async function run() {
+    // Run in subprocess so that handle exhaustion does not affect this process
+    // const sub_run = path.join(__dirname, 'glob_tests', 'run');
```

Comment:
commented-out code (and the comment above can be deleted as well)

Reply:
whoops